blueocean-core-js/src/js/paths/rest.js

```diff
@@ -26,6 +26,14 @@ export default {
         return `${this.apiRoot()}/organizations/${encodeURIComponent(organization)}/pipelines/${pipeline}/runs/${branchStr}`;
     },
 
+    latestRun({ organization, pipeline, branch}) {
```

Comment:
NIT: space after `branch` surprised this didn't fail lint

Reply:
Maybe it did? I didn't look into tests, linting etc yet. Just wanted to get first feedback if the approach was ok or not.